packages/slice-machine/server/src/api/slices/create/index.ts

```diff
@@ -75,39 +64,23 @@ const fromTemplate = async (
   return copyTemplate(env, templatePath, from, sliceName);
 };
 
-export default async function handler({
-  sliceName,
-  from,
-  values,
-}: SliceCreateBody) {
+export default async function handler({ sliceName, from }: SliceBody) {
```

Comment:
I tried to unravel this function last night (side project): I think I'll have to make snap-shot with memfs to avoid breaking it :/

Reply:
Yes so refactoring, by unravel.... I mean work out what's all going on in it (I'd need more displays for that)
I eventually gave up trying to test it because of weird errors